app/livechat/server/config.js

```diff
@@ -272,6 +272,13 @@ Meteor.startup(function() {
 		i18nLabel: 'Office_hours_enabled',
 	});
 
+	settings.add('Livechat_allow_online_agents_outside_office_hours', true, {
+		type: 'boolean',
+		group: 'Livechat',
+		public: true,
+		i18nLabel: 'Allow_Online_Agents_Outside_Office_Hours',
```

Comment:
Would not it be better to enable only if office hours is enabled? Because even if you change the value of this setting, the effect is not applied unless the office hours is enabled. :thinking: